kolibri/plugins/management/assets/src/state/store.js

```diff
@@ -30,33 +29,46 @@ const initialState = {
 };
 
 const mutations = {
-  // modal mutation
+  SET_PAGE_NAME(state, name) {
+    state.pageName = name;
+  },
+  SET_PAGE_STATE(state, pageState) {
+    state.pageState = pageState;
+  },
+
+  // modal mutations
   SET_MODAL(state, modalName) {
     state.pageState.modalShown = modalName;
   },
 
   // class mutations
   ADD_CLASS(state, cl) {
-    state.pageState.classes.push(cl);
-  },
-  DELETE_CLASS(state, id) {
-    state.pageState.classes = state.pageState.classes.filter(cl => cl.id !== id);
+    state.pageState.classrooms.push(cl);
   },
+
   UPDATE_CLASS(state, id, updatedClass) {
-    state.pageState.classes.forEach((cl, index, arr) => {
+    state.pageState.classrooms.forEach((cl, index, arr) => {
       if (cl.id === id) {
         arr[index] = updatedClass;
       }
     });
   },
 
+  DELETE_CLASS(state, id) {
+    state.pageState.classrooms = state.pageState.classrooms.filter(cl => cl.id !== id);
+  },
+
+  DELETE_CLASSROOM_USER(state, id) {
+    state.pageState.classroomUsers = state.pageState.classroomUsers.filter(user => user.id !== id);
+  },
```

Comment:
I'm a little hesitant to have so much logic inside the mutations. Similar with the `UPDATE_CLASS` and `ADD_CLASS` mutations above  - cc @66eli77 

In most other places in the app, this sort of logic lives in the actions, and mutations are dumb setters. Is this deviation from the usual pattern intentional?